test/client/bst-alexa-test.ts

```diff
@@ -20,25 +20,16 @@ describe("BSTAlexa Without Config", function() {
     });
 });
 
-describe("BSTAlexa", async function() {
+describe("BSTAlexa", function() {
     let alexa: BSTAlexa = null;
     let lambdaServer: LambdaServer = null;
 
-    before(async function () {
-        Global.initialize(false);
-        await Global.loadConfig();
-    });
-
     describe("#start()", function () {
         let sandbox: any = null;
 
-        before(async function () {
-            Global.initialize(false);
-            await Global.loadConfig();
-        });
-
         beforeEach(function () {
             sandbox = sinon.sandbox.create();
+            Global.initialize(false, true);
```

Comment:
Is it necessary for this to be called at all with BSTAlexa? If possible, I would like it to not rely on the Global or config.